src/main/java/net/openhft/chronicle/bytes/HexDumpBytes.java

```diff
@@ -1329,6 +1329,240 @@ public boolean lenient() {
         return base.lenient();
     }
 
+    @Override
+    public @NotNull Bytes<Void> append(char ch) throws IllegalStateException {
```

Comment:
why do we need new methods in HexDumpBytes?

Reply:
HexDumpBytes records the bytes written and the structure of how they were written.
This was done for binary writing but not text writing.